apps/keyboard/js/keyboard/input_method_manager.js

```diff
@@ -521,22 +474,24 @@ InputMethodManager.prototype.deactivateIMEngine = function() {
 
 InputMethodManager.prototype.handleEvent = function(evt) {
   this.app.console.info('InputMethodManager.handleEvent()', evt);
-  switch (evt.type) {
```

Comment:
For here, I dont understand very well why we used evt.detail instead of this.app.inputContext. It looks like they have very similar data.